molgenis/molgenis-emx2
# fix: graphql error for i18n of descriptions

- [x] Should fix the data-catalogue-acc when running on 8.140 or higher.
- [x] tested against data from 'acc'

To test:
- create a schema 'catalogue' from template 'data_catalogue' (NOT data_catalogue3'
- upload data provided by brenda into ontologies and catalogue respectively
- should now resolve queries
